handbook/fasse.md

```diff
@@ -194,40 +194,51 @@ data/
 
 ## Step 8. Scratch Space
 
-Scratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. The high-performance storage and resource isolation capabilities of a scratch space enhance performance, streamline workflows, and protect valuable data. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
+A acratch space provides a dedicated area for temporary storage and facilitates efficient data processing. It allows you to perform tasks such as data preprocessing, computationally intensive computations, and analysis, while storing intermediate files and temporary results. It is particularly useful in high-performance computing environments, scientific research, and data-intensive tasks where speed, efficiency, and data management are critical.
 
-We have created a scratch space within our FASSE space: 
+**When to use a scratch space?** Whenever you need a temporary storage for intermediate data during processing or computations, use it to store data that is being actively manipulated or processed but is not needed for long-term storage.
```

Comment:
i think we should also note whenever large amounts of non-permanent storage are needed (example: > 5GB?)

Reply:
Could provide a few examples here:
- creating a large dataset compiled from several analytic datasets. merged data is large but takes time to compile so you want to save it to use for analyses but not take up valuable space on FASSE. instead store the compiled dataset in scratch and access each time you need to run your analysis. if the merged dataset goes unmodified for 90 days it may be deleted, in which case you will need to rerun your script to merge the data.
- for bootstrap analyses it is often quicker to create a set of bootstrap datasets. these datasets individually are large and collectively would overrun the usable space on FASSE. instead, store the temporary datasets in scratch and then have your bootstrap analyses load each individually. delete after use!